build-aux/snap/snapcraft.yaml

```diff
@@ -51,11 +53,13 @@ slots:
     name: com.github.johnfactotum.Foliate
 
 layout:
-  /usr/lib/x86_64-linux-gnu/espeak-ng-data:
-    symlink: $SNAP/usr/lib/x86_64-linux-gnu/espeak-ng-data
+  /usr/lib/$SNAPCRAFT_ARCH_TRIPLET/espeak-ng-data:
+    symlink: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/espeak-ng-data
   # This is a bind instead of a (faster) symlink because mbrola uses LD_PRELOAD
   # with libraries from this directory, which doesn't work with symlinks.
-  /usr/lib/x86_64-linux-gnu/mbrola:
-    bind: $SNAP/usr/lib/x86_64-linux-gnu/mbrola
+  /usr/lib/$SNAPCRAFT_ARCH_TRIPLET/mbrola:
+    bind: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/mbrola
   /usr/share/mbrola:
     symlink: $SNAP/usr/share/mbrola
+  /usr/bin/gjs:
+    symlink: $SNAP/gnome-platform/usr/bin/gjs
```

Comment:
This layout will be added by the `gnome-3-34` extension in the following release of snapcraft, so this won't be needed anymore in the future.